src/sass-builder.js

```diff
@@ -1,11 +1,11 @@
 import fs from 'fs';
 import isEmpty from 'lodash/lang/isEmpty';
-import querystring from 'querystring';
```

Comment:
If you remove `querysring` please remove the dependency from `package.json` as well